src/components/HistoryConversions.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { H2 } from './styledElements/H2';
+import { Button } from './styledElements/Button';
+import { ButtonContainer } from './styledElements/ButtonContainer';
+import { EmptyList } from './styledElements/EmptyList';
+import { Table } from './styledElements/Table';
+import { STORAGE_KEY } from '../utils/localStorageKey';
+
+export const HistoryConversions = props => {
+  const { conversionsList, setConversionsHistory } = props;
+
+  const resetHistory = () => {
+    setConversionsHistory([]);
+    localStorage.setItem(STORAGE_KEY, JSON.stringify([]));
```

Comment:
Jeśli w przeglądarce wyłączona jest obsługa local storage (wystarczy wyłączyć pliki dla domeny), to próba odczytu z local storage spowoduje błąd.

Reply:
J.w.